secure-boot-recovery/boot.conf

```diff
@@ -0,0 +1,17 @@
+[all]
+BOOT_UART=1
+WAKE_ON_GPIO=0
+POWER_OFF_ON_HALT=1
+HDMI_DELAY=0
+
+# SD, USB-MSD, BCM-USB-MSD, Network
+BOOT_ORDER=0xf2541
+
+# Disable self-update mode
+ENABLE_SELF_UPDATE=0
+
+# Select signed-boot mode in the EEPROM. This can be used to during development
```

Comment:
Ah - there's the explanation of the difference between signed boot mode and secure boot mode. You probably what that in the documentation too, not just in one of the config files.